cypress/support/commands.js

```diff
@@ -39,3 +43,51 @@ Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', pas
     password
   });
 });
+
+Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
```

Comment:
Hard-coded values are used for login credentials. It's better to move these to a separate configuration or environment file.